### ISSUE TITLE
Desktop app installer hanging Windows 10/11

### ISSUE THREAD

**Issue by autogptfreak** [Mintplex-Labs/anything-llm#2217]
### How are you running AnythingLLM?

AnythingLLM desktop app

### What happened?

This has happened to me twice. Once on a newly refreshed and updated Windows 11 machine and also on a Windows 10 machine.

In both cases accounts are Windows Standard User accounts (i.e. not Administrator accounts)

On the Windows 10 machine I already had version 1.5.x of the app installed just for my user under the Windows user's AppData directory.
When installing 1.6.4 I chose the upgrade. I watched the uninstalled process spawn and run successfully in Task Manager. Then the install proceeded and got about half way before hanging. CPU activity dropped to zero in Task Manager.

Before that I'd had the same experience on Windows 11. After trying to install it just for my user on a Standard User account, I tried to install it for all users (which requires Administrator login) but the result was the same. Long period of inactivity with zero CPU usage.
I solved it on Win11 by logging into Windows directly with the Administrator account and running the installer for all users. This worked fine!

I haven't solved the Win10 upgrade yet, but I guess the same trick should work, i.e. login with the Administrator account and install for all users.

### Are there known steps to reproduce?

- Run desktop installer for Windows 10 or 11 user that doesn't have admin rights (Standard User).
- Choose either to install just for that user or for all users (both failed for me)

**Comment by autogptfreak** [comment]
Hi @timothycarambat, did you discover anything in your investigation? This totally seems like something that could be helped by provided some info to the user during the install process.

**Comment by timothycarambat** [comment]
Could not ever reproduce (on Home and Server) for windows with both Admin and User account!

**Comment by autogptfreak** [comment]
Ok, that's really strange. Especially since I had a completely clean Windows 11 that I'd just reset and updated. Fingers crossed nobody else has the same problem, or if they do they can try my solution in this thread.

**Comment by mkgai** [comment]
Bringing this thread to life again - installer hang: Windows 11 Pro admin account, installation of new 8.0 desktop application (.exe version, apologies if this comment is out-of-place on git comms).

Didn't uninstall previous version first - wasn't sure if this would remove/change directory of LanceDB filesystem.

Installation hung about 15 minutes without cpu activity before I decided to abort, which actually resolved the problem - sending SIGTERM somehow prompted application to reply with next step ~do you want to download ollama libraries~ and I was able to complete the installation successfully.

Thanks for the great project and ongoing support -

**Comment by blakdog64** [comment]
Got sick of getting the same errors even though I followed all the advice and no one seemed to solve the all users and single user issue with the newer updates - I asked Claude to sort it.....
Claude Code>>>> Here, and it's DONE. The silence was me watching the installer finish; here's the verified end state:

AnythingLLM 1.15.0-r2 installed — exe on disk reads 1.15.0.0, registered in Windows as "AnythingLLM 1.15.0-r2"
1,556 MB / 68,097 files written, installer exited cleanly
RAG untouched — lancedb, documents, everything in Roaming\anythingllm-desktop\storage exactly as it was
What fixed it, in one line each:

The updater's own uninstall step was the deadlock (admin-poisoned files + it detecting its own process name), exactly the known GitHub bug #5018.
So I removed the broken install entirely (killed processes, took ownership, deleted folder + old registry entry) so no uninstall step would run.
Fresh silent install of 1.15 went through in seconds. No dialogs, no loop.
Launch AnythingLLM and your workspaces should load as normal. If anything looks off when it opens, tell me.......

Thanks Claude , my shout tonight!
